.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Continuous Integration
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+permissions: read-all
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    permissions: read-all
+    strategy:
+      fail-fast: false
```

Comment:
Does this apply to each step for a nodejs version or the workflow as a whole? Makes sense to keep 12 running if 10 fails but do we really want to run compile if lint or test failed?

Reply:
> You can allow specific jobs in a job matrix to fail without failing the workflow run. For example, if you wanted to only allow an experimental job with node set to 15 to fail without failing the workflow run.
From: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

So it's about different node versions, not steps, if I understand correctly.

Reply:
Yeah it's about node versions, I agree in principle that a linting error could fail the whole matrix immediately and unit test in a specific Node.js versions shouldn't stop the matrix, but I don't think there's an easy way to do that.